src/cmap/commands.ts

```diff
@@ -465,7 +465,7 @@ export class OpMsgRequest {
 
     // flags
     this.checksumPresent = false;
-    this.moreToCome = options.moreToCome || false;
+    this.moreToCome = command.writeConcern?.w === 0 || false;
```

Comment:
Could we check the options first? 

I am thinking about a runCommand that I want to set fire-and-forget for but is not a command that supports a writeConcern field, that should work right? (then we can use this for the endSessions in client.close())

Reply:
Okay, I've added `moreToCome` as an option back.

Reply:
@durran , what are your thoughts on this?

Reply:
Fine with me.

Reply:
Oh that's not what I meant sorry! Can we check the options for the writeConcern setting and can we check it first before checking the command.

Reply:
from slack: checking w = 0 on options is problematic because all options are inherited, fire-and-forget should only be used by writes so checking the command is correct. `moreToCome` will be out option for setting fire-and-forget for special runCommand cases.